src/entity/Bookmark.ts

```diff
@@ -27,7 +27,7 @@ export class Bookmark {
 
   @ManyToOne(() => BookmarkList, {
     lazy: true,
-    onDelete: 'SET NULL',
+    onDelete: 'CASCADE',
   })
   list: Promise<BookmarkList>;
```

Comment:
I think we always add a seperate property for id as well, so in this case "listId".

But also, maybe it's a bit redundant, I would still prefer this to be called "bookmarkList" rather than "list", but that's perhaps just me.

Reply:
we already have it on line 17 ✔️